### DISCUSSION TITLE
Start dagster dev from python API

### DISCUSSION THREAD

**Discussion by dmsfabiano** [simple-icons/simple-icons#9137] [General]
Is there a way to execute the comparative version of `dagster dev ...` in the cli but as a python API?

Sort of how you can do `uvicorn.run` instead?

I have a local setup python file that boots up all my services and would be nice to do it through python API. I could not find anything in the docs

**Comment by LitoMore** [comment]
Are you asking for a Python package for Simple Icons? Please see https://github.com/sachinraja/simple-icons-py.

In the meantime, why are you posting the same content to another project at https://github.com/dagster-io/dagster/discussions/15631?
